app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -97,7 +97,7 @@ class ThreadController @Inject constructor(
         filterFolder: Folder?,
     ): List<Thread> = withContext(ioDispatcher) {
 
-        fun MutableRealm.keepOldMessagesAndAddToSearchFolder(remoteThread: Thread, searchFolder: Folder) {
+        fun MutableRealm.keepOldMessagesAndAddToSearchFolder(remoteThread: Thread) {
```

Comment:
I don't think this method adds anything to the search folder, is the name still apropriate?

Reply:
True, the `add` part was removed in Octobre 2024.